modules/ezcontent_node/modules/ezcontent_landing_page/config/install/core.entity_form_display.node.landing_page.default.yml

```diff
@@ -3,10 +3,13 @@ status: true
 dependencies:
   config:
     - field.field.node.landing_page.body
+    - field.field.node.landing_page.field_meta_tags
     - field.field.node.landing_page.layout_builder__layout
     - field.field.node.landing_page.layout_selection
     - node.type.landing_page
   module:
+    - content_moderation
```

Comment:
Why do we need to add Content moderation ? Is content moderation part of `Landing page` Content type ?